### ISSUE TITLE
Using ORCID on Setup Fails

### ISSUE THREAD

**Issue by kzscisoft** [FAIRDataPipeline/FAIR-CLI#17]
Need to fix ORCID setup during `fair init`.

**Comment by richardreeve** [comment]
Error trace:
```
(fair-4NTWZjtG-py3.8) (base) bleaberry[~/tmp/demo] fair init
Initialising FAIR repository, setup will now ask for basic info:

Checking for local registry
Local registry found
Remote API URL: https://data.scrc.uk/api/
Local API URL [http://localhost:8000/api/]:
Local API currently offline, would you like to start the server now? [y/N]: y
Successfully connected to local API
Email: richard.reeve@glasgow.ac.uk
ORCID [None]: 0000-0003-2589-8091
Traceback (most recent call last):
  File "/Users/richardr/Library/Caches/pypoetry/virtualenvs/fair-4NTWZjtG-py3.8/bin/fair", line 5, in <module>
    cli()
  File "/Users/richardr/Library/Caches/pypoetry/virtualenvs/fair-4NTWZjtG-py3.8/lib/python3.8/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/Users/richardr/Library/Caches/pypoetry/virtualenvs/fair-4NTWZjtG-py3.8/lib/python3.8/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/Users/richardr/Library/Caches/pypoetry/virtualenvs/fair-4NTWZjtG-py3.8/lib/python3.8/site-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/richardr/Library/Caches/pypoetry/virtualenvs/fair-4NTWZjtG-py3.8/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/richardr/Library/Caches/pypoetry/virtualenvs/fair-4NTWZjtG-py3.8/lib/python3.8/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/Users/richardr/Documents/git/FAIR-CLI/fair/cli.py", line 71, in init
    fair_session.initialise()
  File "/Users/richardr/Documents/git/FAIR-CLI/fair/session.py", line 458, in initialise
    self._global_config = fdp_conf.global_config_query()
  File "/Users/richardr/Documents/git/FAIR-CLI/fair/configuration.py", line 275, in global_config_query
    f"Found entry: {_user_info['given_name']} {_user_info['family_name']}"
UnboundLocalError: local variable '_user_info' referenced before assignment
```

**Comment by kzscisoft** [comment]
Should be fixed in 4534770056c916abff5725a4ca61c0604642504e
